src/components/Modals/WithdrawalStatusModal/WithdrawalStatusModal.tsx

```diff
@@ -0,0 +1,94 @@
+import * as React from 'react'
+import { Button, Close, Radio } from 'decentraland-ui'
+import { t } from 'decentraland-dapps/dist/modules/translation/utils'
+import Modal from 'decentraland-dapps/dist/containers/Modal'
+import { isPending } from 'decentraland-dapps/dist/modules/transaction/utils'
+import { WithdrawalStatus } from '../../../modules/mana/types'
+import { FINISH_WITHDRAWAL_SUCCESS } from '../../../modules/mana/actions'
+import { closeModal } from '../../../modules/modal/actions'
+import { Props } from './WithdrawalStatusModal.types'
+import './WithdrawalStatusModal.css'
+
+export default class WithdrawalStatusModal extends React.PureComponent<Props> {
+  render() {
+    const {
+      name,
+      onClose,
+      metadata,
+      withdrawals,
+      transactions,
+      isLoading,
+      onFinishWithdrawal
+    } = this.props
+    const withdrawal = withdrawals.find(({ hash }) => metadata.txHash === hash)
+    const isTxPending = transactions.some(
+      (tx) =>
+        tx.actionType === FINISH_WITHDRAWAL_SUCCESS &&
+        tx.payload.withdrawal.hash === metadata.hash &&
+        isPending(tx.status)
+    )
+    if (!withdrawal) {
+      return
+    }
+    const { status, amount } = withdrawal
+    const isWithdrawalPending = status === WithdrawalStatus.PENDING
+    const isWithdrawalCheckpoint = status === WithdrawalStatus.CHECKPOINT
+    const isWithdrawalCompleted = status === WithdrawalStatus.COMPLETE
+
+    const handleFinishWithdrawal = () => onFinishWithdrawal(withdrawal)
+    const handleCloseModal = () => closeModal('WithdrawalStatusModal')
```

Comment:
this should be `const handleCloseModal = () => onClose()`

Reply:
Thanks @cazala ! fixed!